SU-SWS/stanford_profile_helper
# D8CORE-3045 Adjusted the margin and padding for people cards

# READY FOR REVIEW

# Summary
- adjusted the margin and padding for people images in cards.

# Need Review By (Date)
- 10/4

# Urgency
- medium

# Steps to Test
1. checkout this branch
1. create a basic page with a teaser as 1 of 3 items in a row
1. reference a profile that has an image
1. ensure the image has equal padding around its sides and top.

![image](https://user-images.githubusercontent.com/7185045/100948694-4c63bc00-34bd-11eb-84cd-e1dc6c4cda58.png)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
